### PR TITLE
Upload or browse dialog

### PR DESCRIPTION
**Pull request by fabrixxm**
This pull request add a colorbox dialog for "upload photo" and "attach file" jot's buttons and for "image" comment's format button.
The user can select a previously uploaded photo or file to add to the post or comment, or upload a new one.

In js/filebrowser.js there is documentation on how call and get response from filebrowser popup.

### CODE REVIEW

File: js/filebrowser.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/**
+ * Filebrowser - Friendica Communications Server
+ * 
+ * Copyright (c) 2010-2013 the Friendica Project
```
Comment (tobiasd) [inline js/filebrowser.js]: not 2015?

Comment (fabrixxm) [inline js/filebrowser.js]: Copy&Paste from LICENSE file :-)

**Comment by tobiasd** [comment]
If there is a footer (i.e. impressum) that is loaded into the popup as well and overlaps with the regular elements of the dialog..

**Comment by fabrixxm** [comment]
@tobiasd see friendica/friendica-addons#291

**Comment by tobiasd** [comment]
thanks!

**Pull request merged**